app/src/components/NavBar.vue

```diff
@@ -68,36 +66,19 @@
   <!-- </header> -->
 </template>
 <script setup lang="ts">
-import { onMounted, watch } from 'vue'
-import { ToastType } from '@/types'
-import { useToastStore } from '@/stores/toast'
+import { onMounted } from 'vue'
 import { logout } from '@/utils/navBarUtil'
-import { useTipsBalance, useWithdrawTips } from '@/composables/tips'
 import IconHamburgerMenu from '@/components/icons/IconHamburgerMenu.vue'
 import IconBell from '@/components/icons/IconBell.vue'
 
-const emits = defineEmits(['toggleSideButton', 'toggleEditUserModal'])
-const { show } = useToastStore()
-const { execute, data: balance, isLoading: balanceLoading, error: balanceError } = useTipsBalance()
-const { execute: withdraw, isLoading: withdrawLoading, error: withdrawError } = useWithdrawTips()
+const emits = defineEmits(['toggleSideButton', 'toggleEditUserModal', 'withdraw', 'getBalance'])
```

Comment:
i think you don't need to get the ballance in the nav bar. 
You can get it in the App.js by watching the state is Auth